### PR TITLE
Implement VariableInfo() and BulkVariableInfo().

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: internal/server/v1/info/golden/bulk_variable_info_test.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+			{
+				[]string{"Count_Person_Female", "Count_Person_Female_AsianAlone", "FertilityRate_Person_Female"},
+				"females.json",
+				false,
+			},
+			{
+				[]string{"IncrementalCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase", "CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase"},
+				"covid.json",
+				false,
+			},
```
Comment (shifucun) [inline internal/server/v1/info/golden/bulk_variable_info_test.go:56]: Can merge all 3 test cases into 1?

File: proto/v1/info.proto [hunk 2]
```diff
@@ -34,3 +35,20 @@ message BulkPlaceInfoRequest {
...
+  string entity = 1;
+  datacommons.StatVarSummary info = 2;
+}
+
+message BulkVariableInfoRequest {
+  repeated string entities = 1;
+}
+
+message BulkVariableInfoResponse {
+  repeated VariableInfoResponse info = 1;
```
Comment (shifucun) [inline proto/v1/info.proto:53]: info -> data ?

File: internal/server/v1/info/golden/bulk_variable_info_test.go (hunk 1 above)
Comment (Spaceenter) [reply internal/server/v1/info/golden/bulk_variable_info_test.go]: done

File: proto/v1/info.proto (hunk 2 above)
Comment (Spaceenter) [reply proto/v1/info.proto]: done

**Pull request merged**